MSFragger-GUI/src/umich/msfragger/cmd/ProcessBuilderInfo.java

```diff
@@ -10,7 +13,11 @@ public ProcessBuilderInfo(ProcessBuilder pb, String name, String fnStdOut,
       String fnStdErr) {
     this.pb = pb;
     this.name = name;
+    final boolean redirectErrorStream =

```

Comment:
This is wrong, will only redirect stderr when stdout is redirected. I'll make another fix.